requirements_dmri.txt

```diff
@@ -0,0 +1,3 @@
+#git+https://github.com/jhlegarreta/tractodata@main
+#git+ssh://git@github.com/fdumais/ismrm_2015_tractography_challenge_scoring.git
+git+ssh://git@github.com/fdumais/scilpy.git@master
```

Comment:
These should not point to our own forks (the first one's original source being my GitHub space).